From bunk@stusta.de Mon Feb 19 16:07:52 2007
From: Adrian Bunk <bunk@stusta.de>
Date: Tue, 20 Feb 2007 01:07:49 +0100
Subject: make kernel/kmod.c:kmod_mk static
To: Andrew Morton <akpm@linux-foundation.org>, Kay Sievers <kay.sievers@vrfy.org>
Cc: linux-kernel@vger.kernel.org, Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20070220000749.GD13958@stusta.de>
Content-Disposition: inline

This patch makes the needlessly global struct kmod_mk static.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 kernel/kmod.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/kernel/kmod.c
+++ gregkh-2.6/kernel/kmod.c
@@ -48,7 +48,7 @@ static struct workqueue_struct *khelper_
 	modprobe_path is set via /proc/sys.
 */
 char modprobe_path[KMOD_PATH_LEN] = "/sbin/modprobe";
-struct module_kobject kmod_mk;
+static struct module_kobject kmod_mk;
 
 /**
  * request_module - try to load a kernel module
